CHANGELOG-1.4.md

```diff
@@ -50,6 +51,7 @@ CHANGELOG - ZIKULA 1.4.x
     - symfony updated to from 2.8.4 to 2.8.8
     - swiftmailer updated from 5.4.1 to 5.4.3
     - components/jquery-ui updated from 1.11.4 to 1.12.0
+    - beberlei/doctrine-extensions updated from 0.3 to 1.0.11
```

Comment:
> If you're still using Paginator, LargeCollections, Phing, PHPUnit or Versionable behaviours available in 0.1–0.3, you're welcome to use 0.3 – but do note, this functionality is now available in Doctrine core, no longer supported in this library, and was removed in 1.0.

:-1: on the update. The current version is only there for BC and subject to removal in 2.0.